### PR TITLE
Convert PresentationSession to PresentationConnection.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: m47/PresentationConnection.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<dl>
+  <dt>PresentationConnection.close()</dt>
+  <dd>Closes the current connection and fires PresentationConnection.onstatechange.</dd>
+  <dt>PresentationConnection.send()</dt>
+  <dd>Sends either binary or text data between a controlling browsing context and a presenting browsing context.</dd>
+</dl>
+
+## Specifications
+
+<http://w3c.github.io/presentation-api/#interface-presentationsession>
```
Comment (mounirlamouri) [inline m47/PresentationConnection.md]: http://w3c.github.io/presentation-api/#interface-presentationconnection

(ie. s/session/connection/)

File: m47/PresentationConnectionAvailableEvent_Constructor.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  <dd>TBD</dd>
+</dl>
+
+## Example
+
+TBD
+
+## Specifications
+
+<http://w3c.github.io/presentation-api/#interface-presentationsessionconnectevent>
```
Comment (mounirlamouri) [inline m47/PresentationConnectionAvailableEvent_Constructor.md:28]: same here, s/session/connection/

File: m47/Presentation.md [hunk 3]
```diff
@@ -12,17 +12,17 @@ The Presentation interface of the [Presentation API](Presentation_API.md) provid
...
 
 <dl>
-  <dt>Presentation.getSession()</dt>
-  <dd>Returns a {{jsxref("Promise")}} that resolves to a <a href="PresentationSession.md">PresentationSession</a> interface.</dd>
-  <dt>Presentation.getSessions()</dt>
-  <dd>Returns a {{jsxref("Promise")}} that resolves to a list of <a href="PresentationSession.md">PresentationSession</a> interfaces.</dd>
+  <dt>Presentation.getConnection()</dt>
+  <dd>Returns a {{jsxref("Promise")}} that resolves to a <a href="PresentationConnection.md">PresentationConnection</a> interface.</dd>
+  <dt>Presentation.getConnections()</dt>
+  <dd>Returns a {{jsxref("Promise")}} that resolves to a list of <a href="PresentationConnection.md">PresentationConnection</a> interfaces.</dd>
```
Comment (mounirlamouri) [inline m47/Presentation.md]: getConnection() and getConnections() have moved to the PresentationReceiver interface in the specification. We do not implement these yet though.

File: m47/PresentationConnectionAvailableEvent_Constructor.md (hunk 2 above)
Comment (jpmedley) [inline m47/PresentationConnectionAvailableEvent_Constructor.md:28]: I'm not sure what you're asking. I've removed the word session in all the text except the spec link. The spec link can't change until the spec does.

**Pull request merged**